### PR TITLE
Cleanup inconsistent placeholders

### PR DESCRIPTION
**Pull request by dniku**
I grew a little tired of `<YOUR CODE >`, so I fixed it course-wide.

### CODE REVIEW

File: week01_intro/crossentropy_method.ipynb [hunk 1]
```diff
@@ -367,13 +367,13 @@
     "\n",
     "for i in range(100):\n",
     "\n",
-    "    %time sessions = [ < generate a list of n_sessions new sessions > ]\n",
+    "    %time sessions = [ <generate a list of n_sessions new sessions> ]\n",
```
Comment (yhn112) [inline week01_intro/crossentropy_method.ipynb:370]: Not consistent

File: week01_intro/crossentropy_method.ipynb [hunk 2]
```diff
@@ -367,13 +367,13 @@
...
     "\n",
     "    states_batch, actions_batch, rewards_batch = zip(*sessions)\n",
     "\n",
-    "    elite_states, elite_actions = <select elite states/actions >\n",
+    "    elite_states, elite_actions = <select elite states/actions>\n",
```
Comment (yhn112) [inline week01_intro/crossentropy_method.ipynb:374]: Not consistent

File: week01_intro/crossentropy_method.ipynb [hunk 3]
```diff
@@ -367,13 +367,13 @@
...
     "\n",
-    "    new_policy = <compute new policy >\n",
+    "    new_policy = <compute new policy>\n",
```
Comment (yhn112) [inline week01_intro/crossentropy_method.ipynb:376]: Not consistent

File: week01_intro/deep_crossentropy_method.ipynb [hunk 4]
```diff
@@ -228,7 +228,7 @@
     "\n",
     "for i in range(100):\n",
     "    # generate new sessions\n",
-    "    sessions = [ < generate a list of n_sessions new sessions > ]\n",
+    "    sessions = [ <generate a list of n_sessions new sessions> ]\n",
```
Comment (yhn112) [inline week01_intro/deep_crossentropy_method.ipynb:231]: Not consistent

File: week01_intro/project_starter_evolution_strategies.ipynb [hunk 5]
```diff
@@ -150,7 +150,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "< tons of your code here or elsewhere >"
+    "<tons of your code here or elsewhere>"
```
Comment (yhn112) [inline week01_intro/project_starter_evolution_strategies.ipynb:153]: Not consistent

File: week02_value_based/seminar_vi.ipynb [hunk 6]
```diff
@@ -483,7 +483,7 @@
     "\n",
     "    # Compute new state values using the functions you defined above.\n",
     "    # It must be a dict {state : float V_new(state)}\n",
-    "    new_state_values = <YOUR CODE HERE >\n",
+    "    new_state_values = <YOUR CODE HERE>\n",
```
Comment (yhn112) [inline week02_value_based/seminar_vi.ipynb:486]: Not consistent

File: week02_value_based/seminar_vi.ipynb [hunk 7]
```diff
@@ -1146,7 +1146,7 @@
     "    Run the policy iteration loop for num_iter iterations or till difference between V(s) is below min_difference.\n",
     "    If policy is not given, initialize it at random.\n",
     "    \"\"\"\n",
-    "    < A WHOLE LOT OF YOUR CODE >\n",
+    "    <YOUR CODE: A WHOLE LOT OF IT>\n",
```
Comment (yhn112) [inline week02_value_based/seminar_vi.ipynb:1149]: Not consistent (remove Caps Locked text)

File: week03_model_free/homework.ipynb [hunk 8]
```diff
@@ -140,9 +140,9 @@
     "        gamma = self.discount\n",
     "        learning_rate = self.alpha\n",
     "\n",
-    "        <YOUR CODE HERE >\n",
+    "        <YOUR CODE HERE>\n",
```
Comment (yhn112) [inline week03_model_free/homework.ipynb:143]: Not consistent (remove "HERE")

File: week03_model_free/homework.ipynb [hunk 9]
```diff
@@ -126,7 +126,7 @@
     "        if len(possible_actions) == 0:\n",
     "            return 0.0\n",
     "\n",
-    "        <YOUR CODE HERE >\n",
+    "        <YOUR CODE HERE>\n",
```
Comment (yhn112) [inline week03_model_free/homework.ipynb:129]: Not consistent (remove "HERE")

File: week03_model_free/homework.ipynb [hunk 10]
```diff
@@ -154,7 +154,7 @@
     "        if len(possible_actions) == 0:\n",
     "            return None\n",
     "\n",
-    "        <YOUR CODE HERE >\n",
+    "        <YOUR CODE HERE>\n",
```
Comment (yhn112) [inline week03_model_free/homework.ipynb:157]: Not consistent (remove "HERE")

File: week03_model_free/homework.ipynb [hunk 11]
```diff
@@ -180,7 +180,7 @@
     "        # agent parameters:\n",
     "        epsilon = self.epsilon\n",
     "\n",
-    "        <YOUR CODE HERE >\n",
+    "        <YOUR CODE HERE>\n",
```
Comment (yhn112) [inline week03_model_free/homework.ipynb:183]: Not consistent (remove "HERE")

File: week03_model_free/homework.ipynb [hunk 12]
```diff
@@ -221,7 +221,7 @@
     "        if len(possible_actions) == 0:\n",
     "            return 0.0\n",
     "\n",
-    "        <YOUR CODE HERE: SEE DOCSTRING >\n",
+    "        <YOUR CODE: SEE DOCSTRING>\n",
```
Comment (yhn112) [inline week03_model_free/homework.ipynb:224]: Not consistent (remove Caps Locked text)

File: week03_model_free/homework.ipynb [hunk 13]
```diff
@@ -508,12 +508,12 @@
...
     "        \"\"\"\n",
-    "        idxes = <randomly generate batch_size integers to be used as indexes of samples >\n",
+    "        idxes = <YOUR CODE: randomly generate batch_size integers to be used as indexes of samples>\n",
     "\n",
     "        # collect <s,a,r,s',done> for each index\n",
-    "        <YOUR CODE >\n",
+    "        <YOUR CODE>\n",
     "\n",
-    "        return np.array( < states > ), np.array( < actions > ), np.array( < rewards > ), np.array( < next_states > ), np.array( < is_done > )"
+    "        return np.array( <states> ), np.array( <actions> ), np.array( <rewards> ), np.array( <next_states> ), np.array( <is_done> )"
```
Comment (yhn112) [inline week03_model_free/homework.ipynb:516]: Not consistent

File: week03_model_free/seminar_qlearning.ipynb [hunk 14]
```diff
@@ -105,7 +105,7 @@
     "        if len(possible_actions) == 0:\n",
     "            return 0.0\n",
     "\n",
-    "        <YOUR CODE HERE >\n",
+    "        <YOUR CODE HERE>\n",
```
Comment (yhn112) [inline week03_model_free/seminar_qlearning.ipynb:108]: Not consistent (remove "HERE")

File: week03_model_free/seminar_qlearning.ipynb [hunk 15]
```diff
@@ -119,9 +119,9 @@
     "        gamma = self.discount\n",
     "        learning_rate = self.alpha\n",
     "\n",
-    "        <YOUR CODE HERE >\n",
+    "        <YOUR CODE HERE>\n",
```
Comment (yhn112) [inline week03_model_free/seminar_qlearning.ipynb:122]: Not consistent (remove "HERE")

File: week03_model_free/seminar_qlearning.ipynb [hunk 16]
```diff
@@ -133,7 +133,7 @@
     "        if len(possible_actions) == 0:\n",
     "            return None\n",
     "\n",
-    "        <YOUR CODE HERE >\n",
+    "        <YOUR CODE HERE>\n",
```
Comment (yhn112) [inline week03_model_free/seminar_qlearning.ipynb:136]: Not consistent (remove "HERE")

File: week03_model_free/seminar_qlearning.ipynb [hunk 17]
```diff
@@ -159,7 +159,7 @@
     "        # agent parameters:\n",
     "        epsilon = self.epsilon\n",
     "\n",
-    "        <YOUR CODE HERE >\n",
+    "        <YOUR CODE HERE>\n",
```
Comment (yhn112) [inline week03_model_free/seminar_qlearning.ipynb:162]: Not consistent (remove "HERE")

File: week03_model_free/seminar_qlearning.ipynb [hunk 18]
```diff
@@ -214,12 +214,12 @@
...
     "    for t in range(t_max):\n",
     "        # get agent to pick action given state s.\n",
-    "        a = <YOUR CODE >\n",
+    "        a = <YOUR CODE>\n",
     "\n",
     "        next_s, r, done, _ = env.step(a)\n",
     "\n",
     "        # train (update) agent for state s\n",
-    "        <YOUR CODE HERE >\n",
+    "        <YOUR CODE HERE>\n",
```
Comment (yhn112) [inline week03_model_free/seminar_qlearning.ipynb:222]: Not consistent (remove "HERE")

File: week04_[recap]_deep_learning/practice_lasagne.ipynb [hunk 19]
```diff
@@ -32,7 +32,7 @@
     "\n",
     "\n",
     "def sum_squares(N):\n",
-    "    return < student.Implement_me() >"
+    "    return <student.implement_me()>"
```
Comment (yhn112) [inline week04_[recap]_deep_learning/practice_lasagne.ipynb:35]: Not consistent

File: week04_[recap]_deep_learning/practice_lasagne.ipynb [hunk 20]
```diff
@@ -225,8 +225,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "inputs = [ < two vectors that my_transformation depends on > ]\n",
-    "outputs = [ < What do we compute (can be a list of several transformation) > ]\n",
+    "inputs = [ <two vectors that my_transformation depends on> ]\n",
```
Comment (yhn112) [inline week04_[recap]_deep_learning/practice_lasagne.ipynb:228]: Not consistent

File: week04_[recap]_deep_learning/practice_lasagne.ipynb [hunk 21]
```diff
@@ -225,8 +225,8 @@
...
+    "outputs = [ <What do we compute (can be a list of several transformation)> ]\n",
```
Comment (yhn112) [inline week04_[recap]_deep_learning/practice_lasagne.ipynb:229]: Not consistent

File: week04_[recap]_deep_learning/practice_lasagne.ipynb [hunk 22]
```diff
@@ -315,9 +315,9 @@
     "# Quest #1 - implement a function that computes a mean squared error of two input vectors\n",
     "# Your function has to take 2 vectors and return a single number\n",
     "\n",
-    "<student.define_inputs_and_transformations() >\n",
+    "<student.define_inputs_and_transformations()>\n",
```
Comment (yhn112) [inline week04_[recap]_deep_learning/practice_lasagne.ipynb:318]: Not consistent

File: week04_[recap]_deep_learning/practice_lasagne.ipynb [hunk 23]
```diff
@@ -315,9 +315,9 @@
...
     "\n",
-    "compute_mse = <student.compile_function() >"
+    "compute_mse = <student.compile_function()>"
```
Comment (yhn112) [inline week04_[recap]_deep_learning/practice_lasagne.ipynb:320]: Not consistent

File: week07_seq2seq/practice_theano.ipynb [hunk 24]
```diff
@@ -583,7 +583,7 @@
     "        translations[0]) is str and len(translations) == len(trans_ix)\n",
     "\n",
     "    # computes levenstein distances. can be arbitrary python code.\n",
-    "    distances = <apply get_distance to each pair of[words, translations] >\n",
+    "    distances = <apply get_distance to each pair of[words, translations]>\n",
```
Comment (yhn112) [inline week07_seq2seq/practice_theano.ipynb:586]: Not consistent

File: week07_seq2seq/practice_theano.ipynb [hunk 25]
```diff
@@ -571,7 +571,7 @@
     "    \"\"\"\n",
     "\n",
     "    # convert words to strings\n",
-    "    words = <restore words(a list of strings) from words_ix >\n",
+    "    words = <restore words(a list of strings) from words_ix>\n",
```
Comment (yhn112) [inline week07_seq2seq/practice_theano.ipynb:574]: Not consistent

**Comment by dniku** [comment]
All comments are addressed.

**Pull request merged**